WeatherService/Presentation/MenuService.cs

```diff
@@ -0,0 +1,13 @@
+﻿
+namespace WeatherService.Presentation
```

Comment:
Always try to consider using scoped name spaces it will help writing cleaner code with less indentation. 

```suggestion
namespace WeatherService.Presentation;
```